storage/src/persistent/codec.rs

```diff
@@ -182,7 +182,7 @@ impl BincodeEncoded for () {}
 
 /// Create number from a bytes
 ///
-/// ```rust
+/// ```rust,no_run
```

Comment:
Had to disable this one because it tries to load a library that is not available and fails.